Makefile

```diff
@@ -582,3 +582,16 @@ push-image-%: registry-login build-image-$$*
 		echo pushed "${IMAGE_ORG}/$*:${VERSION}"; \
 		echo; \
 	) 2>&1 | sed 's/^/  /'
+
+## openshift release openshift-ci operator
+openshift-ci-test-build:
+	@rm -rf "manifests"
+	@rm -rf "metadata"
+	@mkdir -p "manifests";
+	@mkdir -p "metadata";
+	@cp -a "config/docker/addon-operator-bundle.Dockerfile" "addon-operator-bundle.Dockerfile";
+	@cp -a "config/olm/addon-operator.csv.tpl.yaml" "manifests/addon-operator.csv.yaml";
+	@tail -n"+3" "config/deploy/addons.managed.openshift.io_addons.yaml" > "manifests/addons.crd.yaml";
+	@tail -n"+3" "config/deploy/addons.managed.openshift.io_addonoperators.yaml" > "manifests/addonoperators.crd.yaml";
+	@tail -n"+3" "config/deploy/addons.managed.openshift.io_addoninstances.yaml" > "manifests/addoninstances.crd.yaml";
+	@cp -a "config/olm/annotations.yaml" "metadata";
```

Comment:
Would you mind putting these files under `.cache/<folder>` ?

We try to put all the build auto-generated things there to make sure they don't conflict/collide with things top-level in the future.